Display/features/steps/calculating_tubing_descriptors.py

```diff
@@ -0,0 +1,20 @@
+from behave import *
+
+import server
+
+@given("we can use sensor data")
+def step_impl(context):
+
```

Comment:
Is this space in pep8?  Because there's spaces in pep8, and I don't think this space is there, and there's two spaces between functions.

Reply:
I added the blank based on "Method definitions inside a class are surrounded by a single blank line."  Granted this isn't in a class.  Does "method definition" in this context mean the `def` line, or the whole function block?
Actually looking at PEP8 again, looks like I was supposed to have 2 blank lines: "Surround top-level function and class definitions with two blank lines."

Reply:
yeah, these aren't inside classes, but the 'def' marks the top.  Even so, either way, there isn't a space between the definition line and the contents of the function, but you can put comments in there if you'd like.